[JSON manual] json contains (#63)
CUBRID/cubrid-manual: en/sql/function/json_fn.rst

      '2'
      '3'
      NULL

JSON_CONTAINS_PATH
===================================

.. function:: JSON_CONTAINS_PATH (json_doc, one/all, json path [, json path] ...)

  The **JSON_CONTAINS_PATH** function verifies whether the the given paths exist inside the json_doc.

>   1 is returned with one/all argument given as 'all' only if all given paths exist inside the json_doc, 0 is returned otherwise. In case one/all argument is 'one', 1 is returned if at least one json path exists inside the json_doc, 0 otherwise.

Reviewer: Please rephrase:
```suggestion
  When one/all argument is 'all', all given paths must exist to return 1. Returns 0 otherwise.
  When one/all argument is 'one', it returns 1 if any given path exists. Returns 0 otherwise.
```

- A json scalar contains another json scalar if they have the same type (their JSON_TYPE () are equal) and are equal. As an exception, json integer can be compared and equal to json double (even if their JSON_TYPE () evaluation are different).
- A json array contains a json scalar or a json object if any of json array's elements contains the json_nonarray.
- A json array contains another json array if all the second json array's elements are contained in the first json array.
- A json object contains another json object if, for every (key2, value2) pair in the second object, there exists a (key1, value1) pair in the first object with key1=key2 and value2 contained in value1.
  Otherwise the json element is not contained.

  Returns whether doc2 is contained in root json element of doc1 if no json path argument is given.
  Returns NULL if any argument is NULL.
  An error occurs if any argument is invalid.

.. code-block:: sql

    SELECT JSON_CONTAINS ('"simple"','"simple"');
::

      json_contains('"simple"', '"simple"')
    =======================================
                                          1

.. code-block:: sql

>     SELECT JSON_CONTAINS ('["not important", "important"]','"important"');

Reviewer: should be more readable with names shorter than "importantx"; some examples can get very long.

      '2'
      '3'
      NULL

JSON_CONTAINS_PATH
===================================

.. function:: JSON_CONTAINS_PATH (json_doc, one/all, json path [, json path] ...)

>   The **JSON_CONTAINS_PATH** function verifies whether the the given paths exist inside the json_doc.

Reviewer: ```suggestion
  The **JSON_CONTAINS_PATH** function verifies whether the given paths exist inside the json_doc.
```
